app/frontend/styles/_review_warning.scss

```diff
@@ -0,0 +1,15 @@
+.review-warning {
```

Comment:
We generally try to use `app-*` for our custom BEM-y CSS modules, as recommended by the Design System:

```suggestion
.app-review-warning {
```

Reply:
Good to know - updated.